src/DIRAC/WorkloadManagementSystem/Client/JobStatus.py

```diff
@@ -4,13 +4,16 @@
 from __future__ import absolute_import
 from __future__ import division
 from __future__ import print_function
+
+
 __RCSID__ = "$Id$"
 
+from DIRAC.ResourceStatusSystem.PolicySystem.StateMachine import State, StateMachine
```

Comment:
:sob: 
```
(base) [chaen@pceplbc04 DIRAC]$ find . -name 'StateMachine.py'
./Core/Utilities/StateMachine.py
./ResourceStatusSystem/PolicySystem/StateMachine.py
./ProductionSystem/Utilities/StateMachine.py
```
How about taking this opportunity to rationalize all that ? Are there no good standard library for such a standard thing ?

Reply:
The Core/Utilities/StateMachine is not used anywhere. The RSS StateMachine is the only one used/extended, also in LHCb, so we can remove or replace the one in Core. There are of course a few libraries out there, I don't know if it's worthy to look into them... our implementation is IMHO enough for such a simple task.